calculate_average.js

```diff
@@ -0,0 +1,14 @@
+
+const calculateAverage = function calculateAverage(grades) {
+
+  let total = 0;
+  for (let i = 0; i < grades.length; i++) {
+    total += grades[i];
+  }
+
+  let avg = total / grades.length;
```

Comment:
does avg end up as an integer or a float? If all the grades are whole numbers, it might be an integer, but you probably want to force it to do float math